src/utils/mime.ts

```diff
@@ -19,7 +19,7 @@ export const getExtension = (mimeType: string): string | undefined => {
   }
 }
 
-const baseMimes: Record<string, string> = {
+export const baseMimes: Record<string, string> = {
```

Comment:
I think when it exports it, naming `mimes` is better than `baseMimes`. 

```ts
const mimes: Record<string, string> = {
  aac: 'audio/aac',
  // ...
```

The prefix `base` is not necessary. It's easy to understand for users without the `base`.